spec/rails_helper.rb

```diff
@@ -1,3 +1,13 @@
+require 'simplecov'
+SimpleCov.start do
+  add_group 'Config', 'config'
+  add_group 'Controllers', 'app/controllers'
+  add_group 'Libs', 'lib'
+  add_group 'Models', 'app/models'
+  add_group 'Serializers', 'app/serializers'
+  add_group 'Specs', 'spec'
```

Comment:
You can remove the Spec group so that you have a more realistic percentage of covered lines.